doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -2294,7 +2292,7 @@ \subsubsection{Inputs}\label{inputs-13}
 
 \paragraph{Field(s): Surface \textless{}1 thru x\textgreater{} Name}\label{fields-surface-1-thru-x-name}
 
-The remaining fields are used to name the \textbf{\hyperref[buildingsurfacedetailed]{BuildingSurface:Detailed}} objects that are associated with the exterior naturally vented cavity. These are the underlying heat transfer surfaces and are defined elsewhere in the input file. These surfaces should all specify OtherSideConditionsModel as their exterior environment. The input object can currently accommodate up to ten surfaces, but it is extensible by modifying the Energy+.idd entry.
+The remaining fields are used to name the \textbf{\hyperref[buildingsurfacedetailed]{BuildingSurface:Detailed}} objects that are associated with the exterior naturally vented cavity. These are the underlying heat transfer surfaces and are defined elsewhere in the input file. These surfaces should all specify OtherSideConditionsModel as their exterior environment. The input object can currently accommodate up to ten surfaces.
```

Comment:
This object is extensible, not limited to 10 surfaces. Also, the introductory paragraph for this object, SurfaceProperty:ExteriorNaturalVentedCavity, needs editing, it still says "(although if you need to use more than 10 surfaces, then the IDD will need to be extended)."

Reply:
Wow--good catch.  I thought I got all of those but clearly missed this one.  Will be fixed in the next commit.